ci/build.yml

```diff
@@ -37,7 +37,7 @@ suite-web build dev:
   variables:
     ASSET_PREFIX: /suite-web/${CI_BUILD_REF_NAME}/web
   script:
-    - yarn install --frozen-lockfile --cache-folder .yarn --prefer-offline
+    - yarn install --immutable
```

Comment:
what about `--immutable-cache --check-cache` params? wouldn't it be safer with them?

Reply:
Check cache will always fetch all packages from remote servers so there will be no speed up benefits. Also I think this option is needed only when you are using zero install a commiting your yarn/cache folder to git and we don't do that.

Reply:
Oh, you're right, that's valid for zero install, thanks.